compiler/rustc_interface/src/passes.rs

```diff
@@ -492,16 +491,22 @@ fn write_out_deps(tcx: TyCtxt<'_>, outputs: &OutputFilenames, out_filenames: &[P
 
             // Emit special comments with information about accessed environment variables.
             let env_depinfo = sess.psess.env_depinfo.borrow();
+
+            // We will soon sort, so the initial order does not matter.
+            #[allow(rustc::potential_query_instability)]
```

Comment:
Question [REV1]: this looks a little bit sus. We will sort soon, but the sort below is `sort_unstable`?